src/prophetnet/attention.rs

```diff
@@ -742,7 +742,7 @@ pub(crate) fn compute_relative_buckets(
         (
             num_buckets,
             relative_positions.zeros_like(),
-            inverse_relative_positions.max1(&inverse_relative_positions.zeros_like()),
+            inverse_relative_positions.maximum(&inverse_relative_positions.zeros_like()),
```

Comment:
Could we use the `max_other` function instead to be consistent between `min` and `max` calls?

```suggestion
            inverse_relative_positions.max_other(&inverse_relative_positions.zeros_like()),
```